src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,19 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => {
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
+
+        {' by '}
+
+        <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
+          {comment.email}
+        </a>
+      </div>
+
+      <div className="CommentInfo__body">{comment.body}</div>
```

Comment:
```suggestion
export const CommentInfo = ({ comment }) => {
  const {name,... } = comment;
  
  return (
    <div className="CommentInfo">
      <div className="CommentInfo__title">
        <strong className="CommentInfo__name">{name}</strong>

        {' by '}

        <a className="CommentInfo__email" href={`mailto:${email}`}>
          {comment.email}
        </a>
      </div>

      <div className="CommentInfo__body">{body}</div>
```
use destructuring to avoid repeating the same variable's name here and in all same cases

Reply:
Is it really necessary? I prefer more how i wrote them rather than using destructuring, mostly because it's easier for me to follow where one or the other variable comes from, also if we had more than one prop in the function it can become really confusing. Even our mentor Kyrylo said that both options should be good.